specs/AppLifecycle/Restart/restartApi.markdown

```diff
@@ -0,0 +1,259 @@
+# Restart API
+
+# Contents
+
+- [Restart API](#restart-api)
+- [Contents](#contents)
+- [Background](#background)
+  - [Description](#description)
+- [API](#api)
+  - [Existing API Summary](#existing-api-summary)
+  - [Existing API Details](#existing-api-details)
+  - [New API Details](#new-api-details)
```

Comment:
Is there supposed to be MIDL 3 or something in this file to review? There's details about enum values and stuff, but no clear MIDL section with RtCop output, etc.

Reply:
And what does the rest of the `AppLifecycle` API look like?

Reply:
We'll get the RtCop output in a MIDL section in this.

Reply:
RECOMMEND: Jon to more carefully review before sending review.  Hamza to add MIDL3 for the feature.

Reply:
Done. Only thing not yet in is the RtCop output.